guides/v2.0/extension-dev-guide/build/di-xml-file.md

```diff
@@ -143,7 +143,7 @@ Node Format:
 
 : `<argument xsi:type="number">{numericValue}</argument>`
 
-Acceptable values for this type include: integers, floats, or [numeric strings](http://us3.php.net/is_numeric){:taget="_blank"}.
+Acceptable values for this type include: integers, floats, or [numeric strings](http://us3.php.net/is_numeric){:target="_blank"}.
```

Comment:
Please also complete this change for v2.3.

Reply:
Hello @shrielenee ,

The same file is been included in v2.3 md file.